hieradata/staging.yaml

```diff
@@ -564,3 +564,5 @@ users::pentest_machines:
 
 mongodb::s3backup::backup::s3_bucket: 'govuk-mongodb-backup-s3-staging'
 mongodb::s3backup::backup::s3_bucket_daily: 'govuk-mongodb-backup-s3-daily-staging'
+
+trade_tariff_uri: 'https://tariff-frontend-staging.cloudapps.digital/'
```

Comment:
I'm wondering if this should be namespaced more. Something like `plek_overrides::tariff`? This is probably more a question for someone on infrastructure